plugins/rest/rest_auth.go

```diff
@@ -356,8 +385,8 @@ type clientTLSAuthPlugin struct {
 	Cert                 string `json:"cert"`
 	PrivateKey           string `json:"private_key"`
 	PrivateKeyPassphrase string `json:"private_key_passphrase,omitempty"`
-	CACert               string `json:"ca_cert,omitempty"`
-	SystemCARequired     bool   `json:"system_ca_required,omitempty"`
+	CACert               string `json:"ca_cert,omitempty"`            // Deprecated
+	SystemCARequired     bool   `json:"system_ca_required,omitempty"` // Deprecated
```

Comment:
Will these still work? Should we log something to inform users that have configured these fields that they're deprecated...?

Reply:
These should work if the users haven't specified the new fields in the `services` config. Also added a deprecation log message.